packages/okta-react/test/support/tokens.js

```diff
@@ -0,0 +1,289 @@
+  expiresAt: 1449699930,
+  scopes: ['openid', 'email'],
+  authorizeUrl: 'https://auth-js-test.okta.com/oauth2/aus8aus76q8iphupD0h7/v1/authorize',
+  issuer: 'https://auth-js-test.okta.com/oauth2/aus8aus76q8iphupD0h7',
+  clientId: 'NPSfOkH5eZrTy8PMDlvx'
+};
+
+tokens.modifiedIdToken = 'eyJhbGciOiJSUzI1NiIsImtpZCI6IlU1UjhjSGJHdzQ0NVFicTh6Vk8xUGNDcFhMOHlHNkljb3ZWYTNsYUNveE0i' +
+  'fQ.eyJzdWIiOiIwMHUxcGNsYTVxWUlSRURMV0NRViIsIm5hbWUiOiJTYW1sIEphY2tzb24iLCJnaXZlbl9uYW1lIjoiU2FtbCIsImZhbWlseV9u' +
+  'YW1lIjoiSmFja3NvbiIsInVwZGF0ZWRfYXQiOjE0NDYxNTM0MDEsImVtYWlsIjoic2FtbGphY2tzb25Ab2t0YS5jb20iLCJlbWFpbF92ZXJpZml' +
+  'lZCI6dHJ1ZSwidmVyIjoxLCJpc3MiOiJodHRwczovL2F1dGgtanMtdGVzdC5va3RhLmNvbSIsImxvZ2luIjoiYWRtaW5Ab2t0YS5jb20iLCJub2' +
+  '5jZSI6ImNjY2NjYyIsImF1ZCI6InNvbWVJZCIsImlhdCI6MTQ0OTY5NjMzMCwiZXhwIjoxNDQ5Njk5OTMwLCJhbXIiOlsia2JhIiwibWZhIiwic' +
+  'HdkIl0sImp0aSI6IlRSWlQ3UkNpU3ltVHM1VzdSeWgzIiwiYXV0aF90aW1lIjoxNDQ5Njk2MzMwfQ.lVt8eAGGGUBpyrkTb2aq21wC-d-GEV-SZ' +
+  'b8fCupQheQ4GOUEh4Gu2VzRuqFwORYHp177H6b91r7Z9E4L0RbkCLe_F7BmM3JD-BxziFVzIPzKBDZdkg5M12EWomxTd9n-lyYQuE4yA2lOG_W6' +
+  '6ldl_qLOvGlLTv52mJhOBQxW8ic';
+
+tokens.modifiedIdTokenClaims = {
+  'sub': '00u1pcla5qYIREDLWCQV',
+  'name': 'Saml Jackson',
+  'given_name': 'Saml',
+  'family_name': 'Jackson',
+  'updated_at': 1446153401,
+  'email': 'samljackson@okta.com',
+  'email_verified': true,
+  'ver': 1,
+  'iss': 'https://auth-js-test.okta.com',
+  'login': 'admin@okta.com',
+  'nonce': 'cccccc',
+  'aud': 'someId',
+  'iat': 1449696330,
+  'exp': 1449699930,
+  'amr': [
+    'kba',
+    'mfa',
+    'pwd'
+  ],
+  'jti': 'TRZT7RCiSymTs5W7Ryh3',
+  'auth_time': 1449696330
+};
+
+tokens.standardAccessToken = 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ2ZXIiOj' +
+                              'EsImp0aSI6IkFULnJ2Ym5TNGlXdTJhRE5jYTNid1RmMEg5Z' +
+                              'VdjV2xsS1FlaU5ZX1ZlSW1NWkEiLCJpc3MiOiJodHRwczov' +
+                              'L2xib3lldHRlLnRyZXhjbG91ZC5jb20vYXMvb3JzMXJnM3p' +
+                              '5YzhtdlZUSk8wZzciLCJhdWQiOiJodHRwczovL2xib3lldH' +
+                              'RlLnRyZXhjbG91ZC5jb20vYXMvb3JzMXJnM3p5YzhtdlZUS' +
+                              'k8wZzciLCJzdWIiOiIwMHUxcGNsYTVxWUlSRURMV0NRViIs' +
+                              'ImlhdCI6MTQ2ODQ2NzY0NywiZXhwIjoxNDY4NDcxMjQ3LCJ' +
+                              'jaWQiOiJQZjBhaWZyaFladTF2MFAxYkZGeiIsInVpZCI6Ij' +
+                              'AwdTFwY2xhNXFZSVJFRExXQ1FWIiwic2NwIjpbIm9wZW5pZ' +
+                              'CIsImVtYWlsIl19.ziKfS8IjSdOdTHCZllTDnLFdE96U9bS' +
+                              'IsJzI0MQ0zlnM2QiiA7nvS54k6Xy78ebnkJvmeMCctjXVKk' +
+                              'JOEhR6vs11qVmIgbwZ4--MqUIRU3WoFEsr0muLl039QrUa1' +
+                              'EQ9-Ua9rPOMaO0pFC6h2lfB_HfzGifXATKsN-wLdxk6cgA';
+
+tokens.standardAccessTokenParsed = {
+  accessToken: tokens.standardAccessToken,
+  expiresAt: 1449703529, // assuming time = 1449699929
```

Comment:
Would it make sense to pull that assumed time into a constant and set everything relative to it, to make the intended time difference more explicitly clear?

Reply:
I pulled this file from another repo. We don't actually need real tokens. I am removing it.